### DISCUSSION TITLE
Commands

### DISCUSSION THREAD

**Discussion by Kawazski** [Zrips/CMI#5820] [Q&A]
Can the commands like /cmi warps do the just /warps ? bcs on every command i must write the "cmi" in the command. Thx for help

**Comment by viasnake** [comment]
I know this is a very late answer, but I thought I would respond in case anyone is looking for it.
If you want to remove the labels of all commands implemented in CMI, edit Alias.yml

For example, in the case of /cmi warps, you need to edit as follows:
```
  /warps:
    Enabled: true
    Tab: true
```
By setting Enabled to true, the command will be able to execute the command without the cmi label.

**Comment by mrfloris** [comment]
https://github.com/mrfdev/CMI/blob/master/Resources/FAQ/cmi-commands.md

**Comment by CloudeLecaw** [comment]
i think the discussion can be closd because it was answered allready.
